ctk/idp/src/main/kotlin/org/codice/compliance/web/sso/RedirectSSOTest.kt

```diff
@@ -142,21 +126,17 @@ class RedirectSSOTest : StringSpec() {
 
             // Get response from AuthnRequest
             val response = sendRedirectAuthnRequest(queryParams)
-            BindingVerifier.verifyHttpStatusCode(response.statusCode)
+            val finalHttpResponse =
+                    getServiceProvider(IdpSSOResponder::class).getResponseForRedirectRequest(
+                            response)
+//            SingleSignOnProfileVerifier.verifyBinding(finalHttpResponse)
```

Comment:
✏️ Should probably include a todo comment here indicating why this is commented out and when/how it will be enabled.